### DISCUSSION TITLE
Copy / paste output from GitHub Actions includes a lot of extra content ("noise") on Safari

### DISCUSSION THREAD

**Discussion by thescientist13** [community/community#18462] [Actions]
I think this must have just started happening recently, or maybe since I am using Safari on this laptop, but if you try and copy / paste the text out of a GitHub Actions run, a lot of extra text comes with the paste that makes it really hard to read say if pasting into an issue or pull request.

For example, if I copy / paste in the contents of [the _Build_ step contents here](https://github.com/ProjectEvergreen/wcc/runs/6851175027?check_suite_focus=true)
<img width="1014" alt="Screen Shot 2022-06-12 at 19 05 58" src="https://user-images.githubusercontent.com/895923/173257232-9aad938f-9c75-45b4-819e-cc4463f813ba.png">

This is what I get when pasting in
```sh
Run npm run build
[2](https://github.com/ProjectEvergreen/wcc/runs/6851175027?check_suite_focus=true#step:5:2)
  npm run build
[3](https://github.com/ProjectEvergreen/wcc/runs/6851175027?check_suite_focus=true#step:5:3)
  shell: /bin/bash -e {0}
[4](https://github.com/ProjectEvergreen/wcc/runs/6851175027?check_suite_focus=true#step:5:5)
[5](https://github.com/ProjectEvergreen/wcc/runs/6851175027?check_suite_focus=true#step:5:6)
> wc-compiler@0.3.0 build
[6](https://github.com/ProjectEvergreen/wcc/runs/6851175027?check_suite_focus=true#step:5:7)
> node ./build.js
```

Not sure if it maybe. had to do with the recent release of [GitHub Actions Job Summaries](https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/)?

**Comment by ghostofcodes** [comment]
That should be ok I guess lol, the links are probably pointing you to the steps you're seeing as text in your console that's why you can see the `#step:5:2` and others.

But hey if you didn't experienced this before but only with Safari, you might want to try from another browser like Chrome or Firefox

**Reply by thescientist13** [reply]
Just did a little more testing and on Chrome it didn't happen so updated the discussion title to reflect Safari for now.  I do try to use all browsers on different machines just to be more familiar them as I mainly do web dev.  As such, I am not always signed in to GitHub on all browsers on all machines, as you need to be logged in to access GitHub Actions output logs.

But yeah, I do understand what the extra content is from, but I'm sure if a Safari user (like myself in this case) was trying to copy / paste an error or message from the CI output into a search, Stack Overflow, or some messaging app with a colleague, then all that extra content is definitely gonna be a pain to clean up, especially on longer messages.  Even the rendered markdown is a bit all over the place.  😞

**Reply by ghostofcodes** [reply]
Right, I have a pretty bad experience myself with Safari and a couple of services, I guess Apple didn't got their hands on optimization for devs yet, it's more of a media browser from what I saw. Sorry for that though, sounds like it really sucks.

By the way, Chrome is really a nice browser and I'll always recommend you it compared to any other browser! Let me know if you need anything else :)

**Reply by thescientist13** [reply]
Thanks and yeah, no worries.  I use Chrome on my main dev machine, so it's not a huge impediment to my primary workflow.  Just something I bumped into the other day and thought it would be good to flag for others / GitHub.

**Reply by tomjakubowski** [reply]
For what it's worth I'm seeing this problem occur today in Chrome 114.
